docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -2,16 +2,22 @@
 Getting Started with Galaxy
 ===========================
 
-Galaxy workflow support in Dockstore is new and a preview of
-more full featured support to come.
+Dockstore supports Galaxy workflows and the :doc:`Launch with Galaxy <../../launch-with/galaxy-launch-with>` tutorial elaborates different ways to launch a Galaxy workflow. Please note that Galaxy workflows cannot be launched by the Dockstore CLI.
+
+Galaxy instances are offered by many resources, free and otherwise. Each come with their own advantages and users can make a choice based on their own needs.
```

Comment:
Maybe "many institutions" rather than "many resources". 
The Galaxy instance itself seems like a resource?